src/pages/fundamentals/pages/command-dsl/pages/function-handlers/pages/search-lookup.adoc

```diff
@@ -0,0 +1,69 @@
+[[function-handler-search-lookup]]
+= Search by Lookup
+The search by lookup function handler performs a search over the targeted domain entity by using the search criteria provided as query parameters.
+
+== Usage
+The search by lookup function handler is invoked by ending the command query with `"/_search?fn=lookup"`, using the preceeding path as the domain entity over which to search. Specific implementation details can be found within `DefaultSearchFunctionHandlerLookup`.
+
+.Examples
+[source,java,indent=0]
+[subs="verbatim,attributes"]
+----
+@Label("Choose a Person")
+@Values(url = "/p/person/_search?fn=lookup&projection.mapsTo=code:id,label:name")
+@ComboBox(postEventOnChange = true)
+private Long chosenPersonId;
+
+@Label("Choose a Woman")
+@Values(url = "/p/person/_search?fn=lookup&where=person.sex.eq('female')&projection.mapsTo=code:id,label:name")
+@ComboBox(postEventOnChange = true)
+private Long chosenWomanId;
+
+@Label("Choose a Gender")
+@Values(url = "/p/staticCodeValue/_search?fn=lookup&where=staticCodeValue.paramCode.eq('/gender')")
+@ComboBox
+private String chosenGender;
+----
+
+== Supported Query Parameters
+[cols="2,2,8"]
+|===
+| Name | Required | Description
+
+| `where` | false | A QueryDSL filter statement used to determine whether or not a record of data should be contained in the final result set.
+| `orderby` | false | A string containing the name of the field by which to group and direction to sort the result set by. (e.g. `"field.asc()`) Possible sort directions are: `.asc()`, `.desc()`.
+| `fetch` | false | An integer value representing the total number of records that should be returned.
+| `aggregate` | false | An aggregate query (if applicable)
+| `projection.mapsTo` | true | A key/value comma-separated string with the keys being the fields of `ParamValue`, and the values being the field to map. (e.g. `"code:id,label:name"`)
+|===
+
+== Sorting
+Sorting (ascending or descending) happens in memory using Collections' `comparator` interface on the property configured with `orderby` clause.
```

Comment:
in memory sorting (for staticCodeValue as well as dynamic domain entity) happens only when the orderBy clause does not start with domain entity alias. e.g. if the search url is ../p/person/_search?fn=lookup&where=person.name.eq('abc')&orderBy=status.asc()

thought the wording in the documentation for this can capture that detail.

Reply:
Updated this one pretty significantly, but I think I captured what you were looking for. Please review!